docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -540,11 +540,19 @@ To migrate content from one collection to another, if the collections are parts
 
 #. Copy content from the source (old) collection to the target collection.
 #. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. The deprecation must be released after the copied content has been included in a release of the target collection.
-#. When the next major release comes:
-
-  * remove the module/plugin from the source collection
-  * add ``redirect`` to the corresponding entry in ``meta/runtime.yml``
-  * remove ``removal_version`` from there
+#. When the next major release of the old collection is prepared:
+
+  * remove the module/plugin from the source collection and its symlink stored in ``plugin/modules`` directory if appropriate (mainly when removing from ``community.general`` and ``community.network``)
```

Comment:
Whatever term we use in the main intro (line 543 above), we should use throughout - so either "old collection" everywhere or "source collection" everywhere - I like "old collection" a bit better.
```suggestion
  * remove the module/plugin from the old collection
  * remove the symlink stored in ``plugin/modules`` directory if appropriate (mainly when removing from ``community.general`` and ``community.network``)
```

Reply:
I'm not sure `old` is better than `source` technically BUT, agreed, it more clear